Source/GSThemeDrawing.m

```diff
@@ -816,6 +816,129 @@ - (void) drawStepperCell: (NSCell*)cell
     [self drawStepperDownButton: downRect];
 }
 
+// NSSwitch drawing methods
+
+- (void) drawSwitchBezel: (NSRect)frame
+                forState: (NSControlStateValue)v
+{
+  NSBezierPath *p;
+  NSPoint point;
+  CGFloat radius;
+  NSColor *backgroundColor = [NSColor windowBackgroundColor]; // offColor;
+
+  if (NSControlStateValueOn == v)
+    {
+      backgroundColor = [NSColor selectedControlColor]; // onColor;
+    }
```

Comment:
I would prefer to set eh off colour in an else branch.

Reply:
Indeed.  Will do that.

Reply:
Done